### PR TITLE
Finding Long Factorials in Java

### PR DESCRIPTION
**Pull request by shahina-bano**
Added the code for  Finding Long Factorials in Java for issue #362.

### CODE REVIEW

**Comment by shahina-bano** [comment]
@raksha009

**Comment by shahina-bano** [comment]
@raksha009 have a look

File: Algorithm/Special_algo/readme.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Extra Questions :
 
 * Kahns_algorithm ----> [C++](/Code/C++/Kahns_algorithm.cpp)
-* Multiplying large integer number algorithm ----> [C++](/Code/C++/Multiplying_large_integer_number.cpp)
+* Finding Long Factorials ----> [Java](/Code/Java/Long_Factorial.java)
```
Comment (krishna-NIT) [inline Algorithm/Special_algo/readme.md:4]: rearrange them in alphabetical order

**Comment by krishna-NIT** [comment]
@shahina-bano so the changes, Resolve  commit and  then ping me here

**Comment by shahina-bano** [comment]
@krishna-NIT done.

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM , but
Test Cases are not nicely presented, have a look once

**Pull request merged**